hedera-node/src/main/java/com/hedera/services/state/merkle/MerkleNetworkContext.java

```diff
@@ -98,10 +104,14 @@ public class MerkleNetworkContext extends PartialMerkleLeaf implements MerkleLea
     private ExchangeRates midnightRates;
     @Nullable private Instant consensusTimeOfLastHandledTxn = null;
     private SequenceNumber seqNo;
+    private int nextTaskTodo;
     private long lastScannedEntity;
+    private long seqNoPostUpgrade;
+    private long lastScannedPostUpgrade;
     private long entitiesScannedThisSecond = 0L;
     private long entitiesTouchedThisSecond = 0L;
     private long preparedUpdateFileNum = NO_PREPARED_UPDATE_FILE_NUM;
+    private byte preExistingEntityScanStatus;
```

Comment:
Will be nice to have Javadoc for the new variables.

Reply:
💯